app/models/location.rb

```diff
@@ -26,6 +26,8 @@ class Location < ActiveRecord::Base
 
   scope :regional, ->(ids) { where(region_id: ids) }
 
+  scope :active, -> { where(active: true) }
```

Comment:
It looks like the `default_scope` includes `where(active: true)` (line 25). Is this scope necessary? I could see adding it later when we remove the `default_scope`.